### ISSUE TITLE
Fatal error when work with Codable type

### ISSUE THREAD

**Issue by inekipelov** [Data-swift/ManagedModels#27]
Please, help me understand crash reason
```
@Model 
final class StoredAccess: NSManagedObject {
    var token: AccessToken
    var expires: Date
    var accessLevel: AccessLevel
    var mainAccount: AccountID
    var userID: UserID
    var sip: AccessSIP
    var deviceID: DeviceID
}

struct AccessSIP: Codable {
    var username: String
    var password: String
    var realm: String
}
```

Log in console
> ManagedModels/NSAttributeDescription+Data.swift:97: Fatal error: Unsupported Attribute value type AccessSIP

**Comment by helje5** [comment]
Ah the assertion was an improper fall through.

But I'm also getting such now:
```
CoreData: fault: One or more models in this application are using transformable properties with transformer names that are either unset, or set to NSKeyedUnarchiveFromDataTransformerName. Please switch to using "NSSecureUnarchiveFromData" or a subclass of NSSecureUnarchiveFromDataTransformer instead. At some point, Core Data will default to using "NSSecureUnarchiveFromData" when nil is specified, and transformable properties containing classes that do not support NSSecureCoding will become unreadable.
CoreData: warning: Property 'sip' on Entity 'ToDo' is using nil or an insecure NSValueTransformer.  Please switch to using "NSSecureUnarchiveFromData" or a subclass of NSSecureUnarchiveFromDataTransformer instead.
```

**Comment by helje5** [comment]
Should be fixed in 0.8.4
